test-utils/src/indexer/test_indexer.rs

```diff
@@ -25,11 +25,14 @@ use account_compression::{
     NullifierQueueConfig, StateMerkleTreeConfig,
 };
 use forester_utils::indexer::{
-    AddressMerkleTreeAccounts, AddressMerkleTreeBundle, BatchedTreeProofRpcResult, Indexer,
-    IndexerError, MerkleProof, NewAddressProofWithContext, ProofRpcResult, StateMerkleTreeAccounts,
-    StateMerkleTreeBundle, TokenDataWithContext,
+    Indexer, IndexerError, MerkleProof, NewAddressProofWithContext,
```

Comment:
this test-indexer implementation is ahead of test-indexer in light-client can you move/merge it to/with test indexer in light-client?